src/main/java/com/fasterxml/jackson/databind/ser/SerializerCache.java

```diff
@@ -28,29 +29,36 @@ public final class SerializerCache
      * By default, allow caching of up to 4000 serializer entries (for possibly up to
      * 1000 types; but depending access patterns may be as few as half of that).
      */
-    public final static int DEFAULT_MAX_CACHED = 4000;
```

Comment:
Should we leave the older constant as Deprecated in place, but add new one (old one being assigned value of new one)? Since its public constant, some code might be referencing it (perhaps unlikely but still)